examples/models/data_tables_server.py

```diff
@@ -23,10 +27,14 @@ def __init__(self):
         self.class_filter = None
 
         self.source = ColumnDataSource()
+        # A bit hacky, but now we have an index column that will carry through our slicing
+        self.df = DataFrame.from_dict(ColumnDataSource.from_df(mpg))
+        self.df.set_index('index', inplace=True)
         self.update_data()
-
-        self.document.add_root((self.create()))
-        self.session = push_session(self.document)
+        self.source.on_change('data', self.update_from_table_change)
+        # XXX: I haven't had much luck running the server in this style
+        # self.document.add_root((self.create()))
+        # self.session = push_session(self.document)
```

Comment:
I believe that was my note. It was just a comment on running an app through server. Not the static generation with `python data_tables_server.py`. I've never gotten 
```
self.document.add_root((self.create()))
self.session = push_session(self.document)
```
to properly serve an app.

Should I move this version of the example to `examples/apps`? And is the understanding that most `apps` can still be run with `python <APP_NAME>`?